boom! testing kv_cab-0.4.3 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 1019d5c3932a6f61452223c2d719bc7ad8fbe7ea5159b44c48661f29236d333e
boom! running `docker start -a 1019d5c3932a6f61452223c2d719bc7ad8fbe7ea5159b44c48661f29236d333e`
kablam!    Compiling kv_cab v0.4.3 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 1.13 secs
boom! running `docker rm -f 1019d5c3932a6f61452223c2d719bc7ad8fbe7ea5159b44c48661f29236d333e`
blam! 1019d5c3932a6f61452223c2d719bc7ad8fbe7ea5159b44c48661f29236d333e
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! eff41ba6845bce1822558592dd588d69f65dd47fa120682c4df1e38e5bde3b37
boom! running `docker start -a eff41ba6845bce1822558592dd588d69f65dd47fa120682c4df1e38e5bde3b37`
kablam!    Compiling kv_cab v0.4.3 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 5.51 secs
boom! running `docker rm -f eff41ba6845bce1822558592dd588d69f65dd47fa120682c4df1e38e5bde3b37`
blam! eff41ba6845bce1822558592dd588d69f65dd47fa120682c4df1e38e5bde3b37
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! 01bc7d699070a9743b93365ec7882e57e570c84b9040037a0c5174e9ca8b9b07
boom! running `docker start -a 01bc7d699070a9743b93365ec7882e57e570c84b9040037a0c5174e9ca8b9b07`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/kv_cab-cdc927cc9768fb5c
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/tests-059b5c2b33594ee4
blam! 
blam! running 15 tests
blam! test test_get_float ... FAILED
blam! test test_create ... FAILED
blam! test test_get_int ... FAILED
blam! test test_get_none ... FAILED
blam! test test_get_string ... FAILED
blam! test test_insert_float ... FAILED
blam! test test_insert_int ... FAILED
blam! test test_insert_string ... FAILED
blam! test test_keys ... FAILED
blam! test test_kv_all ... FAILED
blam! test test_multi_instance ... FAILED
kablam! thread 'thread '<unnamed>' panicked at 'Could not create file after retries', src/lib.rs:53
blam! test test_remove ... FAILED
blam! test test_multithread_instance ... FAILED
kablam! <unnamed>' panicked at 'Could not create file after retries', src/lib.rs:53
kablam! thread '<unnamed>' panicked at 'Could not create file after retries', src/lib.rs:53
kablam! thread '<unnamed>' panicked at 'Could not create file after retries', src/lib.rs:53
blam! test test_multithread_instance_insert ... FAILED
blam! test test_remove_none ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test_get_float stdout ----
blam! 	thread 'test_get_float' panicked at 'Could not create file after retries', src/lib.rs:53
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test_create stdout ----
blam! 	thread 'test_create' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_get_int stdout ----
blam! 	thread 'test_get_int' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_get_none stdout ----
blam! 	thread 'test_get_none' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_get_string stdout ----
blam! 	thread 'test_get_string' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_insert_float stdout ----
blam! 	thread 'test_insert_float' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_insert_int stdout ----
blam! 	thread 'test_insert_int' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_insert_string stdout ----
blam! 	thread 'test_insert_string' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_keys stdout ----
blam! 	thread 'test_keys' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_kv_all stdout ----
blam! 	thread 'test_kv_all' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_multi_instance stdout ----
blam! 	thread 'test_multi_instance' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_remove stdout ----
blam! 	thread 'test_remove' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! ---- test_multithread_instance stdout ----
blam! 	thread 'test_multithread_instance' panicked at 'assertion failed: t_2.join().is_ok()', tests/tests.rs:214
blam! 
blam! ---- test_multithread_instance_insert stdout ----
blam! 	thread 'test_multithread_instance_insert' panicked at 'assertion failed: t_2.join().is_ok()', tests/tests.rs:238
blam! 
blam! ---- test_remove_none stdout ----
blam! 	thread 'test_remove_none' panicked at 'Could not create file after retries', src/lib.rs:53
blam! 
blam! 
blam! failures:
blam!     test_create
blam!     test_get_float
blam!     test_get_int
blam!     test_get_none
blam!     test_get_string
blam!     test_insert_float
blam!     test_insert_int
blam!     test_insert_string
blam!     test_keys
blam!     test_kv_all
blam!     test_multi_instance
blam!     test_multithread_instance
blam!     test_multithread_instance_insert
blam!     test_remove
blam!     test_remove_none
blam! 
blam! test result: FAILED. 0 passed; 15 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 01bc7d699070a9743b93365ec7882e57e570c84b9040037a0c5174e9ca8b9b07`
blam! 01bc7d699070a9743b93365ec7882e57e570c84b9040037a0c5174e9ca8b9b07
